src/game/stage/stage.c

```diff
@@ -319,8 +319,14 @@ void Task_GameStage(void)
 #if DEBUG
 #include "game/character_select.h"
         if (gInput & SELECT_BUTTON) {
+            const u32 initialCharacter = CHARACTER_TAILS;
+            const bool32 allUnlocked = TRUE;
             TasksDestroyAll();
-            CreateCharacterSelectionScreen(CHARACTER_TAILS);
+#if (GAME == GAME_SA1)
+            CreateCharacterSelectionScreen(initialCharacter);
```

Comment:
Could just make this a macro to provide/not provide the other param?

Reply:
It's just debug code, I don't think it matters too much?

Reply:
Ah didn't notice!